tests/routers/test_applications.py

```diff
@@ -324,6 +324,15 @@ def test_approve_application_with_external_onboarding(
     assert response.status_code == status.HTTP_303_SEE_OTHER
     assert response.headers["location"] == lender.external_onboarding_url
 
+    # borrower access external onboarding system again
+    response = client.get(
+        f"/applications/uuid/{accepted_application.uuid}/access-external-onboarding", follow_redirects=False
+    )
+    assert response.status_code == status.HTTP_307_TEMPORARY_REDIRECT
```

Comment:
Let's update the redirect to use 303 here, too. It's typically used as a confirmation to a resource update. https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/303